kubernetes/overlays/agreement-email-sender/dev-refactor/configmap.yaml

```diff
@@ -11,9 +11,9 @@ data:
   PEC_SMTP_PORT: "{{PEC_SMTP_PORT}}"
   PEC_SMTP_SECURE: "{{PEC_SMTP_SECURE}}"
 
-  SMTP_ADDRESS: "{{SMTP_ADDRESS}}"
-  SMTP_PORT: "{{SMTP_PORT}}"
-  SMTP_SECURE: "{{SMTP_SECURE}}"
+  SMTP_ADDRESS: "{{PEC_SMTP_ADDRESS}}" # this refers to mock smtp server
+  SMTP_PORT: "{{PEC_SMTP_PORT}}" # this refers to mock smtp server
+  SMTP_SECURE: "{{PEC_SMTP_SECURE}}" # this refers to mock smtp server
```

Comment:
is it possible to use `SMTP_*` instead of `PEC_SMTP_*` variables?
my concern is that generally when we move config from one env to another I do not copy specific variables, but the whole file, and this is easy to miss